style.css

```diff
@@ -541,3 +569,453 @@ section {
   margin-top: 9%;
   position: absolute;
 }
+
+.desk-js-frm {
+  display: none;
+}
+
+.desktop-only,
+.desk-ellipse {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    margin-left: 5%;
+    width: 85%;
+    height: 100%;
+  }
+
+  #nav-section {
+    width: 85%;
+    height: 90px;
+    background: #fff;
+  }
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - The margins of the nav and headline section do not match the Figma design.
  
  ### YOUR DESIGN
   ![image](https://www7.lunapic.com/do-not-link-here-use-hosting-instead/168081658857020455?39542288180)
  
  ### Figma Design
  ![image](https://user-images.githubusercontent.com/92303891/230497148-fe0cb4ec-d3cb-4811-9dd9-10e412e53dfb.png)

- [ ] Your Design looks great :+1:  Please ensure your project is responsive on both mobile and desktop screens. :pray:   Your project has a horizontal overflow which shows your project is not responsive so please make sure you adjust the style of your alignment of elements to remove the horizontal overflow. You can see the video below. :point_down: 
  
  [Screencast from 04-06-2023 09:24:51 PM.webm](https://user-images.githubusercontent.com/92303891/230497178-e0db8840-b5f0-43ea-903b-82b0e85fe548.webm)

Reply:
Hello, @Owusu-Desmond  thanks for the helpful suggestions. Could you please guide me on how to remove the Horizontal overflow? I have tried different ways But I don't seem to succeed with it. I will greatly appreciate your guidance. ( I have tried overflow: hidden on the body and other parent containers but that doesn't seem to solve the issue)